classes/cyclonedx-export.bbclass

```diff
@@ -287,6 +289,8 @@ python do_deploy_cyclonedx() {
 
     d.setVar("PN", save_pn)
 
+    os.makedirs(d.getVar("CYCLONEDX_EXPORT_DIR"), exist_ok=True)
+
```

Comment:
Not required, since `do_deploy_cyclonedx[cleandirs] = "${CYCLONEDX_EXPORT_DIR}"` will already ensure the folder exists and is empty: https://docs.yoctoproject.org/bitbake/bitbake-user-manual/bitbake-user-manual-metadata.html#variable-flags

```suggestion
```

Reply:
let me rerun my test without it. I added because a previous test was failing, but I changed some stuff since.